### ISSUE TITLE
Add more tests for ROP search

### ISSUE THREAD

**Issue by XVilka** [radareorg/radare2#16110]
Currently the feature is poorly tested.
We need more tests, see for example:
- https://github.com/d4em0n/exrop/tree/master/tests
- https://github.com/sashs/Ropper/tree/master/testcases
- https://github.com/sashs/Ropper/tree/master/test-binaries
- https://github.com/pakt/ropc/tree/master/example
- https://github.com/pakt/ropc/tree/master/examples-ropl
- https://github.com/salls/angrop/tree/master/tests
- https://github.com/JonathanSalwan/ROPgadget/tree/master/test-suite-binaries
- https://github.com/0vercl0k/rp/tree/master/bin/bin_tests
- https://github.com/Boyan-MILANOV/ropium/tree/master/tests

**Comment by adwait1-G** [comment]
Hi @trufae ,

I want to take this up. I would like to work on issues related to the [Exploitation](https://github.com/radareorg/radare2/projects/24) project. Would that be okay?

Regards,
Adwaith

**Comment by trufae** [comment]
Of course! What would you be starting with? Any question or topic? I would ask you to join the discord or telegram channels for further chats rather than using gh issues for conversations.
